[ADD] accounting: new audit report page (#17083)
odoo/documentation: content/applications/finance/accounting/reporting/annual-report.rst

templates. Users can :ref:`edit and customize <accounting/annual-report/edition>` content, :ref:`add
sections, pages, or attachments <accounting/annual-report/additional-content>`, and :ref:`export the
report as a PDF <accounting/annual-report/export>` for distribution.

.. note::
   The annual report is stored in the :doc:`Knowledge app <../../../productivity/knowledge>`, under
   the :guilabel:`Private` category in the sidebar tree, making the editable report visible only to
   its creator. It uses the layout of the Knowledge app.

.. _accounting/annual-report/creation:

Create an annual report
=======================

To create an annual report, follow these steps:

#. Go to :menuselection:`Accounting --> Review --> Annual Report` and click :guilabel:`New`.
#. In the :guilabel:`Create an Annual Report` window, enter a :guilabel:`Title` and, if necessary,
   update the start and end :guilabel:`Dates` of the period covered by the report.
#. Update the :guilabel:`Responsibles` field, if needed. Additional users listed can access the
   annual report in the :guilabel:`Shared` category of the sidebar tree in the :ref:`editable

>    <accounting/annual-report/edition>` view.

Reviewer: I guess as soon as an additional user is listed here, the editable report is available to the listed users in the **Shared** category of the sidebar tree, right?

#. Click :guilabel:`Save`.

A :guilabel:`Draft` annual report card is then created in the :guilabel:`Annual Reports` Kanban
view.

.. note::
   Clicking the card opens the annual report in the :guilabel:`Private` category of the sidebar

>    tree, from where the content can be :ref:`edited <accounting/annual-report/edition>`.

Reviewer: ```suggestion
.. note::
   Clicking the card opens the annual report within the Knowledge app view, from where the content can be :ref:`edited <accounting/annual-report/edition>`.
```

Maybe here mention that it's in the Private category if no other Responsibles have been listed, but in the Shared category if more than one is listed.

> .. _accounting/annual-report/edition:

Reviewer: ```suggestion
.. _accounting/annual-report/edit:
```

Just remembered this one!

===================

> Annual/Audit report

Reviewer 2: I understand you have discussed the whole audit/annual report imbroglio.
While I lack the context, I'm not sure I would include both terms in the heading, especially since you only refer to the annual report on the page. You could mention "audit reports" in the introduction paragraph, something like (not sure it's true though)
> Annual reports, also known as audit reports, document (...)

What do you think?

===================

Annual reports document annual financial performance, operational challenges, and organizational
achievements for stakeholder review.

Odoo :ref:`generates annual reports <accounting/annual-report/creation>` from pre-configured
templates. Users can :ref:`edit and customize <accounting/annual-report/edit>` content, :ref:`add
more content or upload PDF files <accounting/annual-report/additional-content>`, and :ref:`export
the report as a PDF <accounting/annual-report/export>` for distribution.

.. note::
   The annual report is edited using the :doc:`Knowledge app <../../../productivity/knowledge>`
   interface from within the Accounting app.

.. _accounting/annual-report/creation:

Create an annual report
=======================

To create an annual report, follow these steps:

#. Go to :menuselection:`Accounting --> Review --> Annual Report` and click :guilabel:`New`.
#. In the :guilabel:`Create an Annual Report` window, enter a :guilabel:`Title` and, if necessary,
   update the start and end :guilabel:`Dates` of the period covered by the report.

> #. Update the :guilabel:`Responsibles` field, if needed.

Reviewer 2: I would specify what the field is used for, something like:

> Update the Responsibles field to let other users access and edit the report.

#. Click :guilabel:`Save`.

A :guilabel:`Draft` annual report card is then created in the :guilabel:`Annual Reports` Kanban
view.

.. note::
   - Clicking the card opens the annual report in the :ref:`editable view
     <accounting/annual-report/edit>`.
   - The annual report is stored in the :doc:`Knowledge app <../../../productivity/knowledge>`,
     under the :guilabel:`Private` category in the sidebar tree, making the editable report visible
     only to its creator. If additional users are added to the :guilabel:`Responsibles` field, the
     annual report is then moved to the :guilabel:`Shared` category, allowing those users to access

>      it as well.

Reviewer 2: The content of this note could fit slightly better under the Edit section?

.. _accounting/annual-report/edit:

Edit an annual report
=====================

.. note::
   To modify the annual report's :guilabel:`Title`, :guilabel:`Dates`, or :guilabel:`Responsibles`,
   go to :menuselection:`Accounting --> Review --> Annual Report`, click the :icon:`fa-ellipsis-v`
   :guilabel:`(vertical ellipsis)` icon of the annual report card, and select :guilabel:`Configure`

>    to display the :guilabel:`Create an Annual Report` window.

Reviewer 2: ```suggestion
.. note::
   To modify the annual report's :guilabel:`Title`, :guilabel:`Dates`, or :guilabel:`Responsibles`,
   click the :icon:`fa-ellipsis-v` :guilabel:`(vertical ellipsis)` icon of the annual report card, and select :guilabel:`Configure`
```
I would move this note to the section above - you can then shorten it as well

  in the :ref:`exported PDF report <accounting/annual-report/export>`.
- :guilabel:`Annexes`: Displays :ref:`additional items
  <accounting/annual-report/additional-content>` included in the report.
- :guilabel:`Supporting Documents`: Allows :ref:`uploaded PDF files
  <accounting/annual-report/additional-content>` to be included in the report.

To remove items from the annual report, drag and drop them into :icon:`fa-trash` :guilabel:`Open the
Trash` in the sidebar, or hover over the item in the main view and click the :icon:`fa-trash`
:guilabel:`(trash)` icon.

.. tip::
   - Drag and drop items in the sidebar or main view to reorder them.
   - To create a new article, hover over a section in the sidebar and click the :icon:`fa-plus`
     :guilabel:`(plus)` icon that appears next to it. The article is then nested under the relevant
     section.
   - To show or hide sub-articles, click :icon:`fa-sitemap` :guilabel:`Show sub-articles` or
     :icon:`fa-sitemap` :guilabel:`Hide sub-articles` in the main view.

.. _accounting/annual-report/additional-content:

Additional content
------------------

> The :guilabel:`Annexes` section contains additional items, such as attestations, customizable

Reviewer 2: I can't find the attestations - and since there is an "Attestation" section, I find it confusing, but I might have missed something?

<img width="275" height="436" alt="Image" src="https://github.com/user-attachments/assets/4adf4dc4-e8d5-4237-b67f-5212f0cc78fe" />

accounting reports, and articles with sections or tables to complete. Review and delete any items
that should not be included in the final report.

To add additional annexes, click :guilabel:`Annexes` in the sidebar, then, in the main view, choose
one of the following options:

- :guilabel:`Load a Template`: Click to select extra items not included by default. Complete the
  sections, disable the :icon:`fa-toggle-on` :guilabel:`(toggle-on)` icons to hide unnecessary
  parts, and click :guilabel:`Load Template`.
- :guilabel:`Add an Article`: Click to create a blank article to complete manually.

In the :guilabel:`Supporting Documents` section, click :guilabel:`Upload files` to include

> additional PDF files in the annual report.

Reviewer 2: Maybe move that info under the "Supporting Documents" bullet point above? Having the info together would be easier, in my opinion.
Then I would rename the "Additional content" heading to "Annexes"?
